banyan/resource_service_rdp.go

```diff
@@ -254,6 +261,8 @@ func resourceServiceInfraRdpRead(ctx context.Context, d *schema.ResourceData, m
 			return diag.FromErr(err)
 		}
 	}
+
```

Comment:
could call set  only if RDPSettings are present, this can avoid setting null for RDPSettings on import.

Reply:
fixed